packages/ui/src/app/pages/Settings/components/SaveChangesButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { MouseEventHandler } from 'react'
+
+import { ButtonPrimary } from '@/common/components/buttons/Buttons'
+import { SaveSymbol } from '../../../../common/components/icons/symbols/SaveSymbol'
+
+const saveChanges = () => {
+  alert('All changes are saved successfully') //Here SaveFunctions comes
+}
```

Comment:
Please find the relevant email notification API and use it here.

Reply:
sorry , I can't find the api.
I think the backend is not ready for email notification yet.